consensus/istanbul/backend/backend.go

```diff
@@ -298,23 +285,23 @@ func (sb *Backend) IsValidator() bool {
 // SendDelegateSignMsgToProxy sends an istanbulDelegateSign message to a proxy
 // if one exists
 func (sb *Backend) SendDelegateSignMsgToProxy(msg []byte) error {
```

Comment:
Could you add a comment explaining why this function is disabled, or remove the function entirely?